com.unity.shadergraph/Editor/Data/Graphs/GraphData.cs

```diff
@@ -25,7 +25,7 @@ namespace UnityEditor.ShaderGraph
     [FormerName("UnityEditor.ShaderGraph.AbstractMaterialGraph")]
     sealed partial class GraphData : JsonObject
     {
-        public override int latestVersion => 2;
+        public override int latestVersion => 3;
```

Comment:
Version is bumped here so we can upgrade concreteVersion to graphVersion for subgraphs.